.github/ISSUE_TEMPLATE/device-request.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Device Request
+about: Request a new device to be supported.
+title: "Device name"
+labels: Device Request
+assignees: ''
+
+---
+
+<!--
+  - Before requesting a device, please make sure to search the open and closed issues for any requests in the past.
+  - Sometimes devices have been requested before but are not implemented yet due to various reasons.
+  - If there are no hits on your for the device, please proceed.
```

Comment:
no hits on your for the device -> no hits for your device

Reply:
Thanks a lot latest commit includes all your suggestions :+1: